newIDE/app/src/PropertiesEditor/index.js

```diff
@@ -141,6 +141,7 @@ type Props = {|
   instances: Instances,
   schema: Schema,
   mode?: 'column' | 'row',
+  keyPrefix?: string,
```

Comment:
See if, instead of adding keyPrefix everywhere you can add a `key={effect.getType()}` so that the whole properties editor component is unmounted/remounted when the effect type is changed.

Reply:
Thanks, easier indeed